cmd/flow-dps-indexer/README.md

```diff
@@ -15,6 +15,7 @@ Usage of flow-dps-indexer:
   -f, --force               force indexing to bootstrap from root checkpoint and overwrite existing index
   -i, --index string        path to database directory for state index (default "index")
   -l, --level string        log output level (default "info")
+  -m, --metrics string      URL on which to expose metrics (no metrics are exposed when left empty)
```

Comment:
This is not really a URL, it's a host plus path. URL would include the protocol.